### PR TITLE
wallet: No taking voting addresses.

### PR DESCRIPTION
**Pull request by JoeGruffins**
grpc_example: Add ImportVotingAccountFromSeed example.

    wallet: No taking voting addresses.

    Prevent methods from getting voting account addresses. These addresses
    should not be sent funds and are intended to be used only for submission
    scripts and signatures.

### CODE REVIEW

**Comment by JoeGruffins** [comment]
The reasoning behind these changes is that addresses from voting account type accounts should not be casually fetched by users and used as payment addresses. This also adds a few tests to the grpc example to ensure importing a voting account and "NextAddress" for these accounts works as expected.

File: wallet/addresses.go [hunk 1]
```diff
@@ -791,6 +806,11 @@ func (w *Wallet) newChangeAddress(ctx context.Context, op errors.Op, persist per
 // policy.
 func (w *Wallet) NewChangeAddress(ctx context.Context, account uint32) (stdaddr.Address, error) {
 	const op errors.Op = "wallet.NewChangeAddress"
+
+	// Imported voting accounts must not be used for change.
+	if err := w.notVotingAcct(ctx, op, account); err != nil {
```
Comment (matheusd) [inline wallet/addresses.go:811]: Maybe this should've been added to `newChangeAddress` instead? It seems there's a call path via grpc that can in fact cause the voting account to be used in a change address, by using the voting account number as change destination in a call to the rpcserver's `ConstructTransaction`: internally, that calls `Wallet.NewUnsignedTransaction` which cashes out to using the unexported `newChangeAddress`.

Comment (JoeGruffins) [reply wallet/addresses.go:811]: Ok, will move it back to the unexported method.

Comment (JoeGruffins) [reply wallet/addresses.go]: Moved.

**Review by jrick — APPROVED** [review]

**Pull request merged**